stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/images/AttachmentGalleryActivity.kt

```diff
@@ -28,7 +31,11 @@ public class AttachmentGalleryActivity : AppCompatActivity() {
         super.onStart()
         val urls = intent.getStringArrayExtra(EXTRA_KEY_URLS)?.toList().orEmpty()
         val currentIndex = intent.getIntExtra(EXTRA_KEY_CURRENT_INDEX, 0)
-        binding.attachmentGallery.provideImageList(this, urls, currentIndex)
+        binding.attachmentGallery.provideImageList(this, urls, currentIndex) {
+            binding.toolBarGroup.isVisible = isFullScreen
+
+            isFullScreen = !isFullScreen
+        }
```

Comment:
Same for this lambda parameter, at first I thought this was something like `ListAdapter#submitList`, a callback after the new data has been set